### PR TITLE
Feature/android support

### PR DESCRIPTION
**Pull request by mlabrum**
Hey,

Thanks for creating this component, I've fixed up the android support :)
- Adds propTypes
- Fixes Android UI being cut off https://github.com/recr0ns/react-native-material-switch/issues/5
- Adds support for toggling the component via updating the `active` prop

Cheers,

### CODE REVIEW

**Comment by mlabrum** [comment]
Good catch,  I'll fix that :)

File: index.js [hunk 1]
```diff
@@ -123,6 +145,12 @@ var MaterialSwitch = React.createClass({
     });
   },
 
+  componentWillReceiveProps: function(nextProps){
+    if(this.state.active !== nextProps.active){
```
Comment (mlabrum) [inline index.js]: Fixed :D thanks!

**Comment by ErkanSensei** [comment]
Can this be merged please?

**Comment by sarovin** [comment]
+1

**Comment by Jazz747** [comment]
+1

**Comment by Jazz747** [comment]
try to merge it as soon as possible please :)

**Comment by Jazz747** [comment]
@recr0ns Could this be merged into master please?

**Pull request merged**